js/popup.js

```diff
@@ -17,6 +17,12 @@ deleteButton.addEventListener('click', async () => {
   })
   }
 });
+
+gobackButton.addEventListener('click', async () => {
+  const isCheckedArray = document.querySelectorAll('.container input');
+  const dropDownBox = document.querySelectorAll('.dropdown');
+  deleteButtonLogic(isCheckedArray, tabGroupsArray, dropDownBox, true)
```

Comment:
I've tested the extension on my browser and when I select multiple rows + enter my data, but with one row empty with no data when I hit Go Back to cancel, the inputs have weird behavior:

<img width="380" alt="image" src="https://user-images.githubusercontent.com/38140593/158492787-cff9074a-4a46-44ca-a0aa-d7d44889c11e.png">

Then, when I attempt to edit these rows the toggling of the input's state does not allow me to input anything:

( I am seemingly in 'edit mode' here )
<img width="377" alt="image" src="https://user-images.githubusercontent.com/38140593/158492867-7a03f8e0-1da3-4114-9d47-820cdda29dba.png">


I recommend adding specific functionality for the `goBackButton` click event instead of using the `deleteButtonLogic` function which seems to do something totally different than simply 'going back'.

Reply:
aw bummer! Thought I could cheat a bit with that go back functionality. Thanks for the update! I'll go ahead and separate the functions  @zacharytyhacz